include/jemalloc/internal/cache_bin.h

```diff
@@ -239,11 +246,15 @@ cache_bin_ncached_get_local(cache_bin_t *bin, cache_bin_info_t *info) {
  * Internal.
  *
  * A pointer to the position one past the end of the backing array.
+ *
+ * Do not call if racy, because both 'bin->stack_head' and 'bin->low_bits_empty'
```

Comment:
`low_bits_empty` should be `low_bits_full` instead?

Reply:
Correct, thank you!